docs/how-to-build.md

```diff
@@ -4,244 +4,232 @@ slug: /how-to-build
- You don't have to preinstall the specified JDK environment, as Gradle detects the JDK version needed and downloads it automatically.
-+ Gravitino uses the Gradle Java Toolchain to detect and manage JDK versions, and it checks the installed JDK by running the `./gradlew javaToolchains` command. See [Gradle Java Toolchain](https://docs.gradle.org/current/userguide/toolchains.html#sec:java_toolchain).
-+ Gravitino excludes all Docker-related tests by default. To run Docker-related tests, make sure you have installed Docker in your environment and either (1) set `skipDockerTests=false` in the `gradle.properties` file (or use `-PskipDockerTests=false` in the command) or (2) `export SKIP_DOCKER_TESTS=false` in the shell. Otherwise, all tests requiring Docker will be skipped.
-+ macOS uses `docker-connector` to make the Gravitino Trino connector work with Docker for macOS. See [docker-connector](https://github.com/wenjunxiao/mac-docker-connector), `$GRAVITINO_HOME/dev/docker/tools/mac-docker-connector.sh`, and `$GRAVITINO_HOME/dev/docker/tools/README.md` for more details.
-+ You can use OrbStack as a replacement for Docker for macOS. See [OrbStack](https://orbstack.dev/). With OrbStack, you can run Gravitino integration tests without needing to install `docker-connector`.
-+ Depending on how you deploy Gravitino, other software used in conjunction with Gravitino may contain known security vulnerabilities.
-:::
-
-## Quick start
+- Linux, macOS or Windows operating system
+- Git
+- JDK 8 (default), 11 or 17 for running Gradle, the build environment for Gravitino.
+- Python 3.8 (default), 3.9, 3.10, or 3.11, for building the Gravitino Python client
+- Docker (optional), for running the integration tests
 
-1. Clone the Gravitino project.
-
-   If you want to contribute to this open-source project, please fork the project on GitHub first. After forking, clone the forked project to your local environment, make your changes, and submit a pull request (PR).
+:::note
+Depending on how you deploy Gravitino, there may be contain known security vulnerabilities
+in other software used in conjunction with Gravitino.
+:::
 
-   ```shell
-   git clone git@github.com:apache/gravitino.git
-   ```
+### Install WSL on Windows
 
-2. Build the Gravitino project. Running this for the first time can take 15 minutes or more.
+If you are building Gravitino on a Linux or macOS machine, you can proceed to the [next section](#install-jdk).
+To build Apache Gravitino on Windows, you need to install the Windows Subsystem for Linux (WSL)
+from Microsoft which provides you a Linux-like environment for development.
+For installing WSL, refer to the [WSL Installation Guide](https://learn.microsoft.com/en-us/windows/wsl/install).
+The following steps assume that you have installed Unbuntu which is the default distribution for WSL.
```

Comment:
Breaking sentatn ces over lines hive seval errors in the above text. "have installed Unbuntu, which is" or just "have installed Unbuntu, the default"

Reply:
comma added.